core/src/main/java/org/openedx/core/utils/TimeUtils.kt

```diff
@@ -210,4 +211,39 @@ object TimeUtils {
         }
         return formattedDate
     }
+
```

Comment:
Please take a look on https://developer.android.com/reference/android/text/format/DateUtils class. It has it's own implementation of some methods, also it use android core localisation. Maybe we can use it